packages/augment-api/package.json

```diff
@@ -36,14 +36,17 @@
   "bugs": "https://github.com/KILTprotocol/sdk-js/issues",
   "homepage": "https://github.com/KILTprotocol/sdk-js#readme",
   "devDependencies": {
-    "@polkadot/api": "^9.0.0",
-    "@polkadot/typegen": "^9.0.0",
```

Comment:
I wouldn't bump these in isolation. what's the reason for that?

Reply:
We are relying on some features, for instance the `AccountId33` type, that were only released in [9.3.1](https://github.com/polkadot-js/api/releases/tag/v9.3.1).

Reply:
Also the runtime calls stuff has been improved in 9.2.x and 9.3.x

Reply:
can we bump all the v9 deps then?

Reply:
Done in https://github.com/KILTprotocol/sdk-js/pull/637/commits/5b808272becd8f3a41680fd995cb85d6aeaa5f54.